### DISCUSSION TITLE
what's the matter with this???

### DISCUSSION THREAD

**Discussion by xhit-ux** [apernet/hysteria#1147] [Q&A]
![error_txt](https://github.com/apernet/hysteria/assets/143007456/dfebd177-42f0-46f8-90d8-157b67a69351)

**Comment by haruue** [comment]
您好。 

您需要为 Hysteria 编写配置文件， 才能启动它。 
关于如何编写配置文件， 我们有中文指南： https://hysteria.network/zh/docs/getting-started/Client/

另外， 如果您是直接双击下载好的程序， 然后在 Windows Terminal 中看到此提示的话， 请注意 Hysteria Core 没有图形化界面（GUI）。 
您可以在这里找到支持图形化界面的第三方客户端程序： https://hysteria.network/zh/docs/getting-started/3rd-party-apps/

**Reply by xhit-ux** [reply]
哦哦好的，谢谢谢谢，刚刚看文档还没看到客户端部分，唐突了，谢谢谢谢
